utilities.ts

```diff
@@ -24,3 +21,11 @@ export const isValidSchema = (schema: Schema, example: Schema) => {
 
     return true;
 };
+
+export const getValidationWithDependencies = (schema: Schema, dependencies: Schema, example: Schema) => {
```

Comment:
This method will be more flexible if it can support multiple dependencies. If we break up the ipshadow more.

Reply:
The addSchema that I'm using from ajv to load the dependencies also supports an array, so I made that change and updated the test!